x-pack/plugins/oss_telemetry/server/lib/get_past_days.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+export const getPastDays = (dateString: string): number => {
```

Comment:
NIT: have you considered using `moment().diff(dateString, 'days')`?

Reply:
If you care about truncating the result, I think `Math.trunc(moment().diff(dateString, 'days', true))` would do the trick :)

Reply:
I was actually thinking to avoid moment here, this is the reason I took this approach